articles/azure-vmware/concepts-avs-hub-and-spoke-integration.md

```diff
@@ -44,15 +48,15 @@ The architecture has the following main components:
 
     -   **PaaS Spoke:** A PaaS Spoke hosts Azure PaaS services using private addressing thanks to [Private Endpoint](../private-link/private-endpoint-overview.md) and [Private Link](../private-link/private-link-overview.md).
 
--   **Azure Firewall:** Acts as the central piece to segment traffic between the Spokes, on-premises, and Azure VMware Solution.
+-   **Azure Firewall:** Acts as the central piece to segment traffic between the Spokes and Azure VMware Solution.
 
 -   **Application Gateway:** Exposes and protects web apps that run either on Azure IaaS/PaaS or Azure VMware Solution virtual machines (VMs). It integrates with other services like API Management.
 
 ## Network and security considerations
 
 ExpressRoute connections enable traffic to flow between on-premises, Azure VMware Solution, and the Azure network fabric. Azure VMware Solution uses [ExpressRoute Global Reach](../expressroute/expressroute-global-reach.md) to implement this connectivity.
 
-On-premises connectivity may use ExpressRoute Global Reach as well, but it is not mandatory.
+On-premises connectivity must use ExpressRoute Global Reach as well in order to be able to communicate the on-premises vSphere environment with AVS since an ExpressRoute Gateway does not provide transitive routing between its connected circuits. 
```

Comment:
```suggestion
On-premises connectivity must use ExpressRoute Global Reach to communicate with the on-premises vSphere environment with Azure VMware Solution since an ExpressRoute gateway does not provide transitive routing between its connected circuits. 
```